python/paddle/fluid/tests/unittests/mkldnn/test_elementwise_sub_mkldnn_op.py

```diff
@@ -108,6 +110,26 @@ def init_axis(self):
         self.axis = 1
 
 
+class TestMKLDNNElementwiseSubOp40(TestMKLDNNElementwiseSubOp):
+
+    def init_input_output(self):
+        self.x = np.random.uniform(0.1, 2, [180, 1]).astype(self.dtype)
+        self.y = np.random.uniform(0.1, 1, [1, 256]).astype(self.dtype)
+
+        # self.x = np.random.uniform(0.1, 2, [1, 180]).astype(self.dtype)
+        # self.y = np.random.uniform(0.1, 1, [256, 1]).astype(self.dtype)

```

Comment:
Some more commented out code to remove. LGTM other than this. Please wait for our inner tests results before merging.

Reply:
```        
self.x = np.random.uniform(0.1, 2, [1, 180]).astype(self.dtype)
self.y = np.random.uniform(0.1, 1, [256, 1]).astype(self.dtype)
```
this code also has problem, please check it. This PR will not fix this case.

Reply:
Those commented code removed.